client/src/elm/Backend/PrenatalEncounter/Utils.elm

```diff
@@ -60,5 +63,11 @@ progressReportInitiatorFromUrlFragmemt s =
                     |> InitiatorNewEncounter
                     |> Just
 
+            else if String.startsWith "patient-record" s then
```

Comment:
@anvmn Worth to add `-` to the end of the `patient-record` just in case?

Reply:
@mariano-dagostino implemented at 5438007